assets/js/modules/analytics-4/components/dashboard/ActivationBanner/index.js

```diff
@@ -82,15 +152,29 @@ export default function ActivationBanner() {
 		}
 
 		return { dismissOnCTAClick: false };
-	}, [ returnToSetupStep, step ] );
+	}, [
+		step,
+		returnToSetupStep,
+		hasSetupBannerError,
+		setupBannerErrors,
+		setupBannerSelectors,
+		dispatch,
+	] );
 
 	if ( ! uaConnected || ga4Connected ) {
 		return null;
 	}
 
 	switch ( step ) {
 		case ACTIVATION_STEP_REMINDER:
-			return <ReminderBanner onSubmitSuccess={ handleSubmit } />;
+			return (
+				<ReminderBanner
+					onSubmitSuccess={ handleSubmit }
+					errors={
+						! isEmpty( setupBannerErrors ) && setupBannerErrors
+					}
+				/>
```

Comment:
Can you add a story with the error notice in `ReminderBanner.stories.js`?

Reply:
Added, thank you!